spec/motion-support/core_ext/object/inclusion_spec.rb

```diff
@@ -31,4 +31,18 @@ def include?(obj)
       0.in?(LifeUniverseAndEverything.new).should == false
     end
   end
+
+  describe "presence_in" do
+    it "should return self when included" do
+      "stuff".presence_in(%w( lots of stuff )).should == "stuff"
+    end
+
+    it "should return nil when not included" do
+      "stuff".presence_in(%w( lots of crap )).should == nil
+    end
+
+    it "should raise ArgumentError when arg does not respond to `include?`" do
+      lambda { 1.presence_in(1) }.should.raise ArgumentError
+    end
+  end
```

Comment:
Ah, yes, you are right.
This should not be in this PR.
I am sorry to make things confusing.

Well, as I wrote at the comment of this PR, I am not sure this whole idea is suitable as a PR or not.
If this is appropriate as a PR, I will re-arrange things and separate this `#in?` part into another PR.

So, can I just let it leave for now?